src/mon/Monitor.cc

```diff
@@ -1184,7 +1184,7 @@ void Monitor::sync_start(entity_inst_t &other, bool full)
 
     store->apply_transaction(t);
 
-    assert(g_conf->mon_sync_requester_kill_at != 1);
+    assert(g_conf->get_val<int64_t>("mon_sync_requester_kill_at") != 1);
```

Comment:
@shinobu-x could you remove the migrated config options from `legacy_config_opts.h`?